src/Compression/LZ4_decompress_faster.cpp

```diff
@@ -24,6 +24,14 @@
 #include <arm_neon.h>
 #endif
 
+#if defined(__BYTE_ORDER__) && (__BYTE_ORDER__ == __ORDER_BIG_ENDIAN__)
+static inline UInt16 LZ4_readLE16(const void* memPtr)
+{
+        const UInt8* p = reinterpret_cast<const UInt8*>(memPtr);
```

Comment:
@nickitat wrong style - should be mem_ptr instead of memPtr.

Reply:
Fixed